packages/collector/src/announceCycle/agentready.js

```diff
@@ -211,13 +214,13 @@ function sendEOLEvent() {
 /**
  * Sends an issue event to the agent when the Node.js version has reached end of life.
  * It will work for non serverless environments where an agent is present.
- * Also, currently the serverless-acceptor does not have support for events.
+ * (At the time of writing, serverless-acceptor does not have support for events.)
```

Comment:
Suggestion: This sounds a bit like the support for serverless environments would be added here.
How about? 

"At the time of writing, serverless-acceptor does not have support for events. See packages/serverless/src/backend_connector.js"

Reply:
Not sure I follow. This is talking about the Instana **backend** service `serverless-acceptor`. Support for ingesting events would need to be added there before we can add it in any of our modules. I'll try to make that more clear.

Reply:
Yes.
packages/serverless/src/backend_connector.js === backend service serverless-acceptor

Reply:
No. 

https://github.ibm.com/instana/backend/blob/develop/serverless-acceptor/src/main/java/com/instana/serverless/ServerlessAcceptorApp.java ===backend service serverless-acceptor

Anyway, I've changed the comment, so this should hopefully be a bit more clear now.